### DISCUSSION TITLE
After upgrade to v24.1.14 lost connection to microinvertors + Display is off

### DISCUSSION THREAD

**Discussion by Vlasta134** [tbnobody/OpenDTU#1665] [Q&A]
Hallo, please help.
OpenDtu mit 1,3" OLED Display für Hoymiles HMS/HMT Wechselrichter (CMT2300A)] - from e-bay.de + 2 x Hoymiles HMS.
After upgrade to v24.1.14 lost connection to microinvertors + Display is off. Downgrade didn't help.
CMT2300A Status - not configured
I couldn't change frequency - no offer - if I change pool interval: The frequency must be set between 860250000 and 923500000 kHz and must be a multiple of 250kHz!
Location Czech
THX
Vlasta

![NoF](https://github.com/tbnobody/OpenDTU/assets/157019317/1f4c9771-b9bd-41be-9c93-05b0026822ee)
![Info](https://github.com/tbnobody/OpenDTU/assets/157019317/54f86f54-5b2d-461c-89f0-b236cca8a80e)

**Comment by D3R-ST3FAN** [comment]
Your CMT is not detected. Most likely you have no working Hardware profile selected.
You might want to re-upload your pin-mapping from prior the update.

**Reply by Vlasta134** [reply]
Thank you very much for your help, I found the old saved config.json and after restoring everything is working.
Vlasta

**Reply by D3R-ST3FAN** [reply]
Glad you are back in business.
